src/xebia_email_signature/signature.py

```diff
@@ -170,6 +170,14 @@ def _get_readable_weekdays(availability: [int]) -> str:
         "logo_url": "https://cdn.xebia.com/assets/logos/academy.png",
         "url": "https://xebia.com/academy",
     },
+    "Xebia | Xpirit": {
+        "default": "#222222",
+        "full_name": "#FF6200",
+        "unit": "#5A5A5A",
+        "link": "#FF6200",
+        "logo_url": "/../static/Xebia_Xpirit_logo.png",
```

Comment:
Please refer to a CDN for the logo image; I do not want the signature to be dependent on the generator.

Reply:
Added to CDN